src/stirling/source_connectors/socket_tracer/http2_trace_bpf_test.cc

```diff
@@ -148,7 +148,7 @@ class ProductCatalogServiceTraceTest
     // The container runner will make sure it is in the ready state before unblocking.
     // Stirling will run after this unblocks, as part of SocketTraceBPFTest SetUp().
     // Note that this step will make an access to docker hub to download the HTTP image.
-    PX_CHECK_OK(server_.Run(std::chrono::seconds{60}));
+    PX_CHECK_OK(server_.Run(std::chrono::seconds{4 * 60}));
```

Comment:
I could be wrong but I believe this timeout is just for waiting for a ready message from the container, and it doesn't actually get killed at the end of the timeout.